admin/common_auth/models.py

```diff
@@ -49,6 +49,10 @@ class MyUser(AbstractBaseUser, PermissionsMixin):
     date_joined = models.DateTimeField(auto_now_add=True)
     confirmed = models.BooleanField(default=False)
     osf_id = models.CharField(max_length=5, blank=True)
+    desk_key = models.CharField(max_length=45, blank=True)
```

Comment:
That's not how OAuth applications are supposed to work, but it's not impossible that Desk did something weird. `key` and `key_secret` should be associated with an app specific to the COS account, with `token` and `token_secret` being specific to a user in the `openscience` group.